components/OffsetBorder/Box.tsx

```diff
@@ -14,7 +14,7 @@ export function Box({
         `relative`,
         `mb-2.5`,
         container ? `ml-2.5` : ``,
-        `mr-2.5`,
+        `mr-5`,
```

Comment:
This margin value is specifically chosen to offset the `-right-2.5` values in the enclosed div. We should leave this as it was at `mr-2.5`